diagnosis/getkuberneteslogs.sh

```diff
@@ -191,6 +165,27 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+#checks if azure-cli is installed   
+requirements
+
+#workaround for SSL interception
+export AZURE_CLI_DISABLE_CONNECTION_VERIFICATION=1 
+export ADAL_PYTHON_SSL_NO_VERIFY=1
+
+#Validate resource-group
+location=$(az group show -n $RESOURCE_GROUP --query location)
+if [ $? -ne 0 ]; then
+        echo "[$(date +%Y%m%d%H%M%S)][ERR] Specified Resource group not found."
+        exit 1
+fi
+
+#Get the master nodes from the resource group
+master_nodes=$(az resource list -g $RESOURCE_GROUP --resource-type "Microsoft.Compute/virtualMachines" --query "[?tags.poolName=='master'].{Name:name}" --output table)
```

Comment:
I think using `az vm list` is more standard. E.x.:
`az vm list -g $rg --query "[?tags.poolName=='master'].{Name:name}" --output tsv`

Output type `tsv` will simplify looping over the list

Reply:
I can make this change myself